src/features/pages/p2p/steps/styles.module.scss

```diff
@@ -0,0 +1,20 @@
+@use 'features/styles/theme/theme-mixins' as *;
+
+.step_item {
+    border-radius: 16px;
+    min-block-size: 216px;
+    box-shadow: 0px 12px 16px -4px rgba(14, 14, 14, 0.08), 0px 4px 6px -2px rgba(14, 14, 14, 0.03);
+    @include breakpoints(tablet) {
+        border-radius: 0;
+        min-block-size: auto;
+        box-shadow: none;
+    }
+}
+.desktop_icon {
+    width: 96px;
+    height: 96px;
```

Comment:
let's pass it in `<Image src={item.icon_src} alt="Icon" width={is_mobile ? 48 : 96} height={is_mobile ? 48 : 96} />`

Reply:
This is not the image size but the image's wrapper size. Image sizes are generic.